src/index.tsx

```diff
@@ -5,12 +5,12 @@ import { BrowserRouter } from "react-router-dom";
 import { ApolloProvider } from "@apollo/client";
 
 import App from "./App";
+import "./i18n";
 import { client } from "./apolloClient";
-import GlobalStyle from "./assets/styles/global-styles";
+import { worker } from "./mocks/browser";
```

Comment:
Eslint suggested this change. We can double check if it's working when we don't use required.